perf/stability/mysql/templates/client.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: mysql-client
+  namespace: default
+spec:
+  template:
+    metadata:
+      labels:
+        app: mysql-client
+    spec:
+      containers:
+      - args: # We sleep first and will execute mysql cli later. This ensure the cli command will send traffic to mysql server after iptable sidecar traffic injection finishes.
+        - sleep
+        - "30000"
```

Comment:
I think this uses seconds not ms

Reply:
yes. That's intentional to let the client sleep forever. We exercise connetivity in setup.sh now.